### DISCUSSION TITLE
关于Mod管理功能的提议

### DISCUSSION THREAD

**Discussion by SakuraSayaka** [Meloong-Git/PCL#3878] [讨论]
1.筛选出所有需要更新的Mod单独列出的功能（方便看哪些Mod更新了）
2.屏蔽所选Mod更新的功能（用于某些Mod不想更新的情况）
3.优化一下更新提示？直接把更新后的版本列出 而不是需要鼠标指上去（方便看Mod更新到了哪个版本）
4.更新Mod时增加一个确认，并在批量更新时列出更新的Mod以及数量（现在的批量更新有时候点了就后悔了，以及更新完之后还得去找自己更新了什么）
5.参考一些文件管理软件，用其他方法（例如一个隐藏的文件）给Mod存储标签/备注，例如类别、中文名等。可以方便检索、分类以及避免修改Mod名称导致的一些逆天代码崩溃。还可以考虑增加一些关联记忆功能，例如在下载Mod时自动记录前置Mod是什么。

以上都是我认为是在使用体验上影响程度比较高的提议，下面这条写的跟目前的功能就没太大关系了，做成单独的软件似乎也是不错的主意。

6.基于5的情况下还可以深度开发一下Mod管理方面的能力，例如增加库（存储Mod）和实例（实际用的整合包，包括客户端/服务端）的功能，例如把库中符合条件的Mod加入（链接/复制）到实例中，假设在符合版本的前提下库中有A(Client) B(Server) C(Both)三个Mod，自动把AC加入到客户端中，BC加入到服务端中。这样的好处是在对整合包的修改时只需要对库进行操作，对实例的影响都是自动化的，极大程度减少了在不同文件夹倒腾来倒腾去的时间。写到这里其实我想到这种功能也可以不限于Mod，像配置文件，脚本等内容也一直是整合时需要来回复制的大头，不仅可以减少整合时的繁琐操作，如果可以使用链接功能的话还能极大程度减少游戏占用的空间。

**Comment by allMagicNB** [comment]
- #679
- #277

**Comment by Deep-Dark-Forest** [comment]


**Reply by allMagicNB** [reply]
不是，你别乱说

**Reply by Deep-Dark-Forest** [reply]
> 不是，你别乱说

哎抱歉我没查重
